llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: llc -filetype=obj -o %t -mtriple=powerpc-aix-ibm-xcoff < %s
```

Comment:
Does this test need a `REQUIRES` directive, or does the `XCOFF` folder already have one? (As noted before, I'm not particularly familiar with LLVM IR/llc, so I don't know if this comment is actually applicable)

Reply:
Thanks for remind. You are right. We need llc that supports `powerpc-aix-ibm-xcoff`